readme.md

```diff
@@ -0,0 +1,45 @@
+# WP Factories
+
+> Helper to easily create Wordpress taxonomy/custom post type.
+
+## Usage
+
+```
+composer require studiometa/wp-factories
+```
+
+In your wordpress theme:
+
+functions.php
+```php
+use Studiometa\WP\Factories\Post_Type;
+
+/**
+ * Register post types
+ *
+ * @return void
+ */
+public function register_post_types() {
```

Comment:
I think we can remove the class format here to keep the example as straightforward as possible: 

```php
function register_post_types() {
    // ...
}

add_action( 'init', 'register_post_types' );
```